src/apps/sections/services.py

```diff
@@ -104,6 +105,8 @@ async def manage_section_state(
     if weight_involved:
         if stock_weight is None:
             raise ServiceException("Stock weight was not provided! ")
+        print("wow")
+        print(multiplier, stock_weight)
         section_object.available_weight -= multiplier * stock_weight
         section_object.occupied_weight += multiplier * stock_weight
 
```

Comment:
just as mentioned in the comment: https://github.com/mmyszak999/WarehouseMS/pull/18/files#r1652029233